python/heppy_patch/heppyMvBadChunks.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
```

Comment:
another script with the same name exists, I fear that having two is messy, especially if we want to avoid patching and moving scripts by hand every time